it/test/utils/WelshMessagesSpec.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils
+
+import helpers.ComponentSpecBase
+
+class WelshMessagesSpec extends ComponentSpecBase {
```

Comment:
We already have one as script:
income-tax-view-change-frontend/check_messages.sh
we would need to decide to use one of these I guess

Reply:
The difference between this and the script is that this one is run automatically with the other tests, so it will always be checked.

Also, the script currently appears to be flagging comments as well, which is a bit odd

Reply:
I think we can drop this old script, its was more of aesthetic choice rather then logical one to create this script;